src/imshow.jl

```diff
@@ -9,6 +9,15 @@ If working in the REPL, the function tries to choose the encoding
 based on the current display size. The image will also be
 downsampled to fit into the display (using `restrict`).
 """
+function imshow(
+        io::IO,
+        img::AbstractArray{<:Colorant, 3},
```

Comment:
```suggestion
        img::AbstractArray{<:Colorant},
```
Why not just match the `show` method & allow any N here? 1 & 2 should go to more specific methods anyway

Reply:
Thanks for pointing it out. Now it also works for nd `imshow`

```julia
julia> Gray.(rand(2, 16, 2, 2))
2×16×2×2 Array{Gray{Float64},4} with eltype Gray{Float64}:
[:, :, 1, 1] =
████████████████████████████████
████████████████████████████████

[:, :, 2, 1] =
████████████████████████████████
████████████████████████████████

[:, :, 1, 2] =
████████████████████████████████
████████████████████████████████

[:, :, 2, 2] =
████████████████████████████████
████████████████████████████████
```